Dockerfile

```diff
@@ -141,7 +142,8 @@ RUN source /etc/default/evm && \
 RUN source /etc/default/evm && \
     cd ${SUI_ROOT} && \
     yarn install && \
-    yarn run build
+    yarn run build && \
+    rm -rvf node_modules
```

Comment:
Unfortunately, the node_modules will still exist in one of the parent images, I think.  Would be nice if there was a way to make it not exist entirely.